webpack/profiles/production.js

```diff
@@ -9,7 +9,7 @@ const CleanWebpackPlugin = require("clean-webpack-plugin");
 const configValidator = require("../../config/validator");
 const path = require("path");
 
-function makeConfig(projectConfig) {
+function makeConfig(projectConfig, profileName) {
     const profileVariables = projectConfig.profiles.production;
```

Comment:
now we can write this - projectConfig.profiles[profileName]

It could move us to make function makeConfig more abstract and this prepares us to next level of refactoring  )